src/Catty/ViewController/Upload/UploadViewController.swift

```diff
@@ -56,46 +39,46 @@ class UploadViewController: UIViewController {
 
     private var zipFileData: Data?
```

Comment:
I think we do not need any zip data here since that is handled by the uploader service.